src/components/Chat/BottomSheetAttachment.js

```diff
@@ -70,7 +70,7 @@ const BottomSheetAttachment = React.forwardRef((props, ref) => (
             <Text style={styles.textCard}>Send a File</Text>
           </View>
           {props.isLoadingUploadFile ? (
-            <ActivityIndicator size="small" color={COLORS.white2} />
+            <ActivityIndicator size="small" color={COLORS.white} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color of the `ActivityIndicator` has been updated from `COLORS.white2` to `COLORS.white`. As with the previous comments, please ensure that this color change is intentional and does not negatively impact the visibility or accessibility of the component.


<!-- This is an auto-generated comment by OSS CodeRabbit -->